### ISSUE TITLE
Retropie w/Moonlight Doesn't Connect Due to Warning: SSL Verification error :: ok Error

### ISSUE THREAD

**Issue by nathan-hook** [LizardByte/Sunshine#3326]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the latest beta/pre-release?

This issue is present in the latest pre-release

### Describe the Bug

When attempting to connect my raspberry pi retropie via moonlight to sunshine, sunshine logs out an error:
```
Warning: SSL Verification error :: ok
Info: /CN=NVIDIA GameStream Client -- denied
```

TL;DR
I returned from a trip last week and moonlight (on the raspberry pi retropie) would connect to sunshine (the april release), but only for about 2 minutes, and then the raspberry pi retropie would get kicked off from sunshine.

I was thinking that some configuration problem went wrong between the two, uninstalled both moonlight and sunshine, reinstalled them both, and then the raspberry pi retropie wouldn't connect to sunshine at all, and the above error is printed when sunshine is set to debug/verbose logging.

FWIW, I am guessing that the certificate for the /CN=NVIDIA GameStream Client has expired, but I am unsure of how to test that theory out.  From looking at the certificates on the raspberry pi retropie under `/etc/ssl/certs/` it looks like all those certificates were installed in 2022, meaning my retropie was installed around that same time, a perfect amount off time for certificates like NVIDIA GameStream Client to start expiring.

Any thought on how to verify if that particular certificate has expired?

Thank you for your time.

### Host Operating System

Windows

### Operating System Version

Windows 10 Home, 19045.5011

### Architecture

amd64/x86_64

### Sunshine commit or version

Version v2024.1021.153031

### Package

Windows - installer (recommended)

### GPU Type

Nvidia

### GPU Model

NVIDIA GeForce RTX 3060

### GPU Driver/Mesa Version

531.09

### Capture Method

None

### Relevant log output

```shell
Warning: SSL Verification error :: ok
Info: /CN=NVIDIA GameStream Client -- denied
```

**Comment by ReenigneArcher** [comment]
Disable encryption, your client likely doesn't support it yet.

**Comment by cgutman** [comment]
This isn't the full stream encryption (there's a different message for that one). It's a warning from the HTTPS certification validation that should be supported by all clients.

**Comment by nathan-hook** [comment]
The problem exists between the keyboard and chair.  Yay?

When I first remember installing moonlight and sunshine, I didn't remember having to specifically run the `moonlight pair xxx.xxx.xxx.xxx` command by hand.  But perhaps I did, well, anyways, after running the `moonlight pair` command by hand.  Everything started to work as expected.

Here is the exact raspberry pi retropie command I ran:
```
/opt/retropie/supplementary/moonlight/moonlight.sh pair 192.168.50.229
```

Sorry for taking your time, but thank you for the replies.

**Comment by ReenigneArcher** [comment]
Thanks for posting the solution

**Comment by robert-w-gries** [comment]
Posting my Mac Mini solution to this problem here as this Github issue comes up on Google. I upgraded the default openssh available on Mac by using Homebrew:

```
brew install openssh
```

After restarting the Moonlight client on my mac, autodiscovery finally worked  and I was able to connect to the sunshine server.

Note: The process of installing Homebrew also installed Xcode Command Line Tools, so it's possible that the solution was just to install Xcode CLT.
